ci-operator/step-registry/cucushift/installer/check/gcp/disktype-control-plane/cucushift-installer-check-gcp-disktype-control-plane-ref.metadata.json

```
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/check/gcp/disktype-control-plane/cucushift-installer-check-gcp-disktype-control-plane-ref.yaml",
```

```suggestion
	"path": "cucushift/installer/check/gcp/disktype/control-plane/cucushift-installer-check-gcp-disktype-control-plane-ref.yaml",
```

Once that, we can move `cucushift/installer/check/gcp/disktype-compute` to `cucushift/installer/check/gcp/disktype/` as a sub-folder.

---

@jianli-wei I did not see the change here ?